packages/ui/components/proposal/ProposalYourVote.tsx

```diff
@@ -1,5 +1,27 @@
-export interface ProposalYourVoteProps {}
+import clsx from 'clsx'
 
-export const ProposalYourVote = ({}: ProposalYourVoteProps) => {
-  return <p>placeholder</p>
+export type YourVoteProps = {
+  variant: 'abstain' | 'pending' | 'no' | 'yes'
+}
+
+export const ProposalYourVote = ({ variant }: YourVoteProps) => {
+  return (
+    <div
+      className={clsx(
+        'flex relative justify-center items-center py-1 px-4 w-20 text-sm font-medium rounded-full',
+        variant === 'abstain'
+          ? 'text-text-secondary bg-primary'
```

Comment:
```suggestion
          ? 'text-text-secondary bg-background-secondary'
```

using the new color system :D